app/src/main/java/tk/zedlabs/wallportal/viewmodel/PostViewModel.kt

```diff
@@ -1,15 +1,14 @@
 package tk.zedlabs.wallportal.viewmodel
 
-import androidx.lifecycle.ViewModel
 import androidx.lifecycle.LiveData
+import androidx.lifecycle.ViewModel
 import androidx.lifecycle.viewModelScope
 import androidx.paging.LivePagedListBuilder
 import androidx.paging.PageKeyedDataSource
 import androidx.paging.PagedList
 import tk.zedlabs.wallportal.models.WallHavenResponse
-import tk.zedlabs.wallportal.repository.PopularDataSource
 import tk.zedlabs.wallportal.repository.PopularDataSourceFactory
-import tk.zedlabs.wallportal.repository.PostDataSource
+import tk.zedlabs.wallportal.repository.PostDataSource.Companion.PAGE_SIZE
```

Comment:
these changes should also be replicated in `PopularViewModel`

Reply:
DRY dont repeat yourself. Why create 2 same constants. better use common